src/spinners/ClimbingBoxLoader.jsx

```diff
@@ -74,13 +74,15 @@ class Loader extends React.Component {
 Loader.propTypes = {
   loading: PropTypes.bool,
   color: PropTypes.string,
-  size: PropTypes.number
+  size: PropTypes.number,
+  sizeunit: PropTypes.string
 };
 
 Loader.defaultProps = {
   loading: true,
   color: '#000000',
-  size: 15
+  size: 15,
+  sizeunit: ''
```

Comment:
should this be `px`?

Reply:
I wasn't 100% sure on this. All of the other Loaders had a `px` string at the end, but for this loader, the size goes to `font-size: ${this.props.size};`, which had no `px` at the end.